source/assets/js/helpers/module.js

```diff
@@ -12,7 +12,7 @@ class EstaticoModule {
 	* @param  {object} options - The options passed as data attribute in the Module
 	*/
 	constructor($element, _defaultData, _defaultOptions, data, options) {
-		this.name = this.constructor.name.toLowerCase();
+		this.name = this.constructor.name;
```

Comment:
I definitely like the approach from #32 and #33. However the `data-` attributes management remains open, and the approach from @backflip (`[data-' + this.name.toLowerCase() + '="bla"']`) seems rather error prone. How about having `this.name` and `this._name`/`this.dataName` for these 2 purposes (I know, my naming skills are really bad… feel free to propose another one).

This has moreover impact on linting configuration and how we would have to specify defaults in `head.js`: we would need to use PascalCased object properties instead of "camelCased". Note that many times `head.js` is used as the configuration point for backend developers, so having proper naming conventions there is important. @backflip would it be fine for you to change linting rules to accept as well PascalCased object properties?